mantle/cmd/kolet/kolet.go

```diff
@@ -369,13 +376,23 @@ func runReboot(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func runHttpd(cmd *cobra.Command, args []string) error {
+	port := args[0]
+	path := args[1]
```

Comment:
I think it's going to be common to want to specify the path but not the port.  Let's use regular positionals like `kolet httpd --port 9001  --path /path/to/mydata` so that one can omit one or both of them too and have them default to `8000` and `.` or so?

Reply:
That'd make sense.